apps/extension/package.json

```diff
@@ -25,6 +25,7 @@
     "react-dom": "^18.2.0",
     "react-native-web": "^0.18.10",
     "react-scripts": "^5.0.1",
+    "redux-state-sync": "^3.1.4",
```

Comment:
I think we don't need this dependency for `extension` if it used in `libs/ui`

Reply:
there is an error, if I delete it
<img width="667" alt="image" src="https://user-images.githubusercontent.com/35599200/220906248-c0739643-3fdb-43c4-bbce-b57d174e0a0c.png">